index.js

```diff
@@ -1794,7 +1794,7 @@ module.query = function(context, entity, criteria, callback) {
   }
   if (criteria && typeof criteria !== 'function') {
     url += module.criteriaToString(criteria) || ''
-    url = url.replace(/'/g, '%27').replace(/=/, '%3D').replace(/</, '%3C').replace(/>/, '%3E')
+    url = url.replace(/'/g, '%27').replace(/=/, '%3D').replace(/</, '%3C').replace(/>/, '%3E').replace(/\&/g, '%26');
```

Comment:
Please replace all of this special casing with `encodeURIComponent`:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent